manifests/istio-control/istio-discovery/files/gen-istio.yaml

```diff
@@ -828,6 +829,7 @@ data:
         traffic.sidecar.istio.io/excludeOutboundPorts: "{{ annotation .ObjectMeta `traffic.sidecar.istio.io/excludeOutboundPorts` .Values.global.proxy.excludeOutboundPorts }}"
       {{- end }}
         traffic.sidecar.istio.io/kubevirtInterfaces: "{{ index .ObjectMeta.Annotations `traffic.sidecar.istio.io/kubevirtInterfaces` }}"
+      revision: {{ valueOrDefault .Values.revision "" }}
```

Comment:
I think we don't need a new field here, the sidecar injector already knows what revision this is

Reply:
Where?  On my install from master the injector is istiod.  The istiod-global pod has a label `version: global` and I have requested that @istio/wg-environments-maintainers add `istio.io/rev: global`.  But there is no `env.name` = `valueFrom.fieldRef` to get it into istiod.

It might be better to tell istiod its Revision instead of putting it here.  Shall I modify this PR to do that?

Reply:
Fixed.